.gitignore

```diff
@@ -96,3 +96,5 @@ buck-out/
 
 # Bundle artifact
 *.jsbundle
+
+client/flutter/lib/generated
```

Comment:
I think that in general, it makes sense to ignore the generated files since they can easily be regenerated. But, this will add an additional step to the build process when building for release. Eventually, build-for-release should be automated with fastlane, but until then, I'm wary of changing the process since we've established it well with WHO so far.

As an interim step, what would we think about keeping the generated files in the repo, but still accepting this change to add the l10n generation in our CI? That way, the CI will still fail on this line if a mistake is made:
 https://github.com/WorldHealthOrganization/app/blob/c44b7de0cbb511097aedde398c44d02f9eb0a2d6/.github/workflows/flutter-build.yml#L34

cc @advayDev1

Reply:
I agree with you Hunter on both points.